packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -58,17 +55,12 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
   // peerstore migration
   await deleteOldPeerstorePreV036(beaconPaths.peerStoreDir, logger);
 
-  let dbMetrics: null | ReturnType<typeof createDbMetrics> = null;
   // additional metrics registries
-  const metricsRegistries = [];
-  if (options.metrics.enabled) {
-    dbMetrics = createDbMetrics();
-    metricsRegistries.push(dbMetrics.registry);
-  }
+  const metricsRegistries: Registry[] = [];
   const db = new BeaconDb({
     config,
     controller: new LevelDbController(options.db, {logger: logger.child(options.logger.db)}),
-    metrics: dbMetrics?.metrics,
+    metrics: options.metrics.enabled,
```

Comment:
So now `createDbMetrics()` is called in BeaconDb, how are they then wired to the exposed metrics server?